### PR TITLE
User class and console features added

### PR DESCRIPTION
**Pull request by StarCodes2**

### CODE REVIEW

File: .gitignore
Comment (BLESSEDEFEM) [inline .gitignore:1]: It was nicely put

File: console.py
Comment (BLESSEDEFEM) [inline console.py:1]: The code was well written and comprehensible for a visiting programmer

File: models/engine/file_storage.py
Comment (BLESSEDEFEM) [inline models/engine/file_storage.py:1]: A beautiful storage code you have here

File: models/user.py
Comment (BLESSEDEFEM) [inline models/user.py:1]: Every line is written as expected
Nothing to add

File: tests/test_models/test_user.py
Comment (BLESSEDEFEM) [inline tests/test_models/test_user.py:1]: Great lines of python code

**Review by StarCodes2 — COMMENTED** [review]
Everything looks good

**Comment by StarCodes2** [comment]
Look fine

**Pull request merged**